doc/api/errors.md

```diff
@@ -889,6 +889,11 @@ provided.
 Encoding provided to `TextDecoder()` API was not one of the
 [WHATWG Supported Encodings][].
 
+<a id="ERR_EVAL_ESM_CANNOT_PRINT"></a>
+### `ERR_EVAL_ESM_CANNOT_PRINT`
+
+`--print` cannot be used with ESM input.
```

Comment:
Dumb question, but why this limitation?

Reply:
TLA modules complete with `undefined` to mitigate the case where the last evaluated expression is a promise.

Reply:
Not a dumb question at all.